src/features/components/molecules/deriv-products-hero/products-heading.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react'
+import { Localize } from 'components/localization'
+import Typography from 'features/components/atoms/typography'
+import { TString } from 'types/generics'
+
+const ProductsHeading: FC<{ title: TString[] }> = ({ title }) => {
```

Comment:
We are not using React.FC, please read documentation 🙏 
You can use React.PropsWithChildren or ⬇️ 

```suggestion
const ProductsHeading = ({ title }: { title: TString[] }) => {
```

Reply:
hey man i saw some usage in our code base its okay
ok man I will change thanks